wiki/release/release-process.md

```diff
@@ -311,9 +306,25 @@ The vote should be open for at least 72 hours (3 days). Votes from Pulsar PMC me
 will be considered binding, while anyone else is encouraged to verify the release and
 vote as well.
 
-If the release is approved here, we can then proceed to next step.
+If the release is approved here, we can then proceed to next step. Otherwise, we should repeat the previous steps and prepare another candidate release to vote.
```

Comment:
```suggestion
If the release is approved here, we can then proceed to the next step. Otherwise, you should repeat the previous steps and prepare another candidate release to vote.
```
Better keep the pronoun consistent and use the second-person pronoun throughout the guide. It can also be omitted in step-by-step instructions.